src/components/partner/partner.repository.ts

```diff
@@ -270,4 +270,70 @@ export class PartnerRepository extends DtoRepository(Partner) {
         )
       );
   }
+
+  async listPartnersOfAllUserProjects(
+    session: Session,
+    { filter, ...input }: PartnerListInput,
+    scopeSensitivityMap: Partial<Record<ScopedRole, Sensitivity | undefined>>
+  ) {
+    return this.db
+      .query()
+      .match([
+        [
+          node('project'),
+          relation('out'),
+          node('', 'Partnership'),
+          relation('out'),
+          node('node', 'Partner'),
+          ...(filter.userId && session.userId
+            ? [
+                relation('out', '', 'organization', { active: true }),
+                node('', 'Organization'),
+                relation('in', '', 'organization', { active: true }),
+                node('user', 'User', { id: filter.userId }),
+              ]
+            : []),
+        ],
+      ])
+      .apply(matchPropsAndProjectSensAndScopedRoles(session, undefined, true))
+      .subQuery((sub) =>
+        sub
+          .with('sensitivity')
+          .return([
+            `apoc.convert.fromJsonMap('${JSON.stringify(
+              scopeSensitivityMap
+            )}') as sensMap`,
+            `(${rankSens('sensitivity')}) as sens`,
+          ])
+      )
+      .raw(
+        `MATCH (node) WHERE any(x in scopedRoles where x IN keys(sensMap) and sens <= ${rankSens(
+          'apoc.map.get(sensMap, x)'
+        )})`
+      )
```

Comment:
```suggestion
      .raw(
        `
          MATCH (node) WHERE any(x in scopedRoles where x IN keys($sensMap)
              and sens <= ${rankSens('apoc.map.get($sensMap, x)')}
          )
        `, {
          sensMap: scopeSensitivityMap
        }
      )
```

Does this work instead of 
```tsx
`apoc.convert.fromJsonMap('${JSON.stringify(scopeSensitivityMap)}') as sensMap`
```